Common/cpp/Tools/JSScheduler.h

```diff
@@ -13,13 +13,19 @@ class JSScheduler {
  public:
   explicit JSScheduler(
       jsi::Runtime &rnRuntime,
-      const std::shared_ptr<facebook::react::CallInvoker> &jsCallInvoker)
-      : rnRuntime_(rnRuntime), jsCallInvoker_(jsCallInvoker) {}
+      const std::shared_ptr<facebook::react::CallInvoker> &jsCallInvoker,
+      std::function<void(std::function<void(jsi::Runtime &runtime)> &&callback)>
+          runtimeExecutor)
+      : rnRuntime_(rnRuntime),
+        jsCallInvoker_(jsCallInvoker),
+        runtimeExecutor_(runtimeExecutor) {}
   void scheduleOnJS(std::function<void(jsi::Runtime &rt)> job);
 
  protected:
   jsi::Runtime &rnRuntime_;
   const std::shared_ptr<facebook::react::CallInvoker> jsCallInvoker_;
+  std::function<void(std::function<void(jsi::Runtime &runtime)> &&callback)>
```

Comment:
same thing

Reply:
Done in [2efade7](https://github.com/software-mansion/react-native-reanimated/pull/5734/commits/2efade7e68c277b2ab8ca61c31814a910dce2703)